src/ServiceInsight/Framework/Settings/ApplicationConfiguration.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace ServiceInsight.Framework.Settings
+{
+    using System;
+    using System.Configuration;
+
+    public class ApplicationConfiguration
+    {
+        public static bool SkipCertificateValidation => Convert.ToBoolean(ConfigurationManager.AppSettings["SkipCertificateValidation"]);
```

Comment:
I'm not sure a static member is the best way to do this. I'd prefer the app entry point (Program.cs) to reach into app.config and set this value at application start up.

Reply:
Done